server/src/main/java/com/netflix/conductor/server/ServerModule.java

```diff
@@ -1,23 +1,29 @@
 /**
  * Copyright 2016 Netflix, Inc.
- *
+ * <p>
```

Comment:
No changes to the server module should be included in this PR

Reply:
Removed it as it would be merged from another PR #781